pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -511,6 +511,27 @@ def test_mangled(self):
         expected = pd.DataFrame({"a": [0, 0], "b": [1, 1]})
         tm.assert_frame_equal(result, expected)
 
+    def test_agg_namedtuple(self):
+        # GH34422
+        s = pd.Series([1, 1, 2, 2, 3, 3, 4, 5])
+        msg = "func is"
+        with pytest.raises(TypeError, match=msg):
+            s.groupby(s.values).agg(a=pd.NamedAgg(column="anything", aggfunc="min"))
```

Comment:
Thanks @gurukiran07 

The three new tests look very similar, can they be parametrized? See [Parametrizing fixtures and test functions](https://docs.pytest.org/en/latest/parametrize.html if it's not something you're familiar with

Reply:
Thank you, I edit accordingly.